xx.md

```diff
@@ -0,0 +1,89 @@
+# NIP-XX
+
+## Peer-to-peer Order events
+
+`draft` `optional`
+
+## Abstract
+
+Peer-to-peer (P2P) platforms have seen an upturn in recent years, while having more and more options is positive, in the specific case of p2p, having several options contributes to the liquidity split, meaning sometimes there's not enough assets available for trading. If we combine all these individual solutions into one big pool of orders, it will make them much more competitive compared to centralized systems, where a single authority controls the liquidity.
+
+This NIP defines a simple standard for peer-to-peer order events, which enables the creation of a big liquidity pool for all p2p platforms participating.
+
+## The event
+
+Events are [Parameterized Replaceable Events](https://github.com/nostr-protocol/nips/blob/master/01.md#kinds) and use `38383` as event kind, a p2p event look like this:
+
+```json
+[
+  "EVENT",
+  "RAND",
+  {
+    "id": "84fad0d29cb3529d789faeff2033e88fe157a48e071c6a5d1619928289420e31",
+    "pubkey": "dbe0b1be7aafd3cfba92d7463edbd4e33b2969f61bd554d37ac56f032e13355a",
+    "created_at": 1702548701,
+    "kind": 38383,
+    "tags": [
+      ["d", "ede61c96-4c13-4519-bf3a-dcf7f1e9d842"],
+      ["k", "sell"],
+      ["f", "VES"],
+      ["s", "pending"],
+      ["amt", "0"],
+      ["fa", "100"],
```

Comment:
Robosats has orders with ranges, where the taker decides the amount, can we optionally add a 3rd string?  ["fa", "100", "1000"]

Reply:
No ranged order will stay `["fa", "100"]`

Reply:
we already have ranges, check the `fa` description on tags section

Reply:
that's right, sorry :smile: